lib/CGameState.cpp

```diff
@@ -3284,45 +3281,154 @@ void CPathfinder::initializeGraph()
 			for(size_t k=0; k < out.sizes.z; ++k)
 			{
 				curPos = int3(i,j,k);
```

Comment:
Nice idea. I wanted to get rid of "curPos" as class member anyway as I want to use evaluateAccessibility outside of initializeGraph (I don't need whole graph on server for path validation). Good reason to do that.